pkg/integration/gitlab.go

```diff
@@ -133,6 +138,7 @@ func (g *GitLab) Data(
 
 	return map[string][]byte{
 		"host":         []byte(g.host),
+		"port":         []byte(strconv.Itoa(g.port)),
```

Comment:
Why having to convert, can you just define it as string

Reply:
I'd rather an error is raised if the input is bad.